app/mailers/credit_note_mailer.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class CreditNoteMailer < ApplicationMailer
+  before_action :ensure_pdf
+
+  def created
+    @credit_note = params[:credit_note]
+    @organization = @credit_note.organization
+    @customer = @credit_note.customer
+
+    return if @organization.email.blank?
+    return if @customer.email.blank?
+
+    I18n.locale = @customer.preferred_document_locale
```

Comment:
This is changing the local for the whole thread, I suggest you use `I18n.with_local do ... end` so that the local is reseted back to :en after the task